### PR TITLE
Reintegrate skeleton

### PR DESCRIPTION
**Pull request by zenine07**
# 

## 🗣 Description

This PR would reintegrate the [cisagov/skeleton-docker](https://github.com/cisagov/skeleton-docker) skeleton into this repository.

This was done by adding the correct skeleton repository as a remote, pulling from its develop branch, and resolving any conflicts shown. 

```console
git remote add skeleton https://github.com/cisagov/skeleton-docker.git
git remote set-url --push skeleton no_push
git pull --allow-unrelated-histories skeleton develop
git status
```

After skeleton was merged, fixed pre-commit issues that showed up.

## 💭 Motivation and Context

When this project was created, it was not started using the [`skeleton` tool](https://github.com/cisagov/development-guide/tree/develop/project_setup#skeleton-tool-) This prevents the [`lineage` GitHub Action](https://github.com/cisagov/action-lineage) from automatically generating PRs to ensure downstream repositories are kept up-to-date.

## 🧪 Testing

Automated tests pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)
- [x] Integrate skeleton

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by zenine07** [comment]
@dav3r Are we able to disable the LGTM analysis: Python on this repo?

**Comment by dav3r** [comment]
> @dav3r Are we able to disable the LGTM analysis: Python on this repo?

Yes, I will disable it for now.  Can you please remind me to re-enable it after this PR is merged?

**Comment by zenine07** [comment]
> Yes, I will disable it for now. Can you please remind me to re-enable it after this PR is merged?

I can, but why does it need to be re-enabled when this isn't a Python project?

**Comment by dav3r** [comment]
> > @dav3r Are we able to disable the LGTM analysis: Python on this repo?
> 
> Yes, I will disable it for now. Can you please remind me to re-enable it after this PR is merged?

I just checked and none of the checks are required for merging yet, so don't worry about LGTM failing for now.

**Comment by dav3r** [comment]
> > Yes, I will disable it for now. Can you please remind me to re-enable it after this PR is merged?
> 
> I can, but why does it need to be re-enabled when this isn't a Python project?

In this case, LGTM will still be checking the requirements files and the Python code in the `tests/` directory.  It's not much, but it doesn't hurt to have it run.

**Comment by zenine07** [comment]
> In this case, LGTM will still be checking the requirements files and the Python code in the `tests/` directory. It's not much, but it doesn't hurt to have it run.

Sounds good. I'll see if I can get it to succeed.

**Comment by dav3r** [comment]
> > In this case, LGTM will still be checking the requirements files and the Python code in the `tests/` directory. It's not much, but it doesn't hurt to have it run.
> 
> Sounds good. I'll see if I can get it to succeed.

If you can't, don't sweat it too much on this PR.  We have seen it behave in weird ways when it is initially integrated to a project, but tends to work after that.

File: src/secrets/quote.txt
Comment (jsf9k) [inline src/secrets/quote.txt]: Is this file being used here?  If not then it should be removed.

File: README.md
Comment (jsf9k) [inline README.md]: We generally do Markdown headings `## Like this ##` with trailing octothorpes.  Can you make this change throughout this file?

I'm also confused as to why the linters aren't checking this file.  @mcdonnnj?

**Review by jsf9k — CHANGES_REQUESTED** [review]
Great work @zenine07!  I requested only a couple of changes.

File: README.md
Comment (mcdonnnj) [reply README.md]: We don't have anything auto-formatting on Markdown files. We would need to expand the `markdownlint` configuration to enforce this rule because it is on the default `consistent`. We should really look at adding a number of rules to the `markdownlint` configuration to better enforce our preferred Markdown style.

Comment (zenine07) [reply README.md]: @mcdonnnj I believe it was automatically enforcing it when I ran precommit for the python-library-skeleton, but it doesn't do it in the skeleton-docker.

Comment (mcdonnnj) [reply README.md]: @zenine07 The default of `consistent` will only flag if you are using different styles. Since you did not merge in the skeleton's README you are consistently using the ATX style (and not the ATX closed style we prefer).

File: src/secrets/quote.txt
Comment (zenine07) [reply src/secrets/quote.txt]: @jsf9k I forget that file every time. Fixed in https://github.com/cisagov/domain-manager-ui/pull/126/commits/26d7bb2d28f6ea8fd1611a00ba0f5b1e9b15c536

File: README.md
Comment (zenine07) [reply README.md]: @jsf9k Made those changes in https://github.com/cisagov/domain-manager-ui/pull/126/commits/98558103e782fa0eb37426c3aa175bf66fa0961c

**Pull request closed without merging**
